src/utils.js

```diff
@@ -0,0 +1,9 @@
+import { APCAcontrast, sRGBtoY } from "apca-w3";
+import { colorParsley } from "colorparsley";
+
+const shouldUseDarkText = (color) => {
+    const testColor = colorParsley(color)
+    return Math.abs(APCAcontrast(sRGBtoY(colorParsley("black")), sRGBtoY(testColor))) >= Math.abs(APCAcontrast(sRGBtoY(colorParsley("white")), sRGBtoY(testColor)))
+}
```

Comment:
Would it be possible to have 2 optional parameters to pass into the function for the dark/light text colours (that defaults to `"black"` and `"white"`)? Our text is not necessarily `#000000` or `#ffffff` (e.g `#262626` ), so would produce inaccurate contrast values.

For example:

```suggestion
const shouldUseDarkText = (color, darkText = "black", lightText = "white") => {
    const testColor = colorParsley(color)
    return Math.abs(APCAcontrast(sRGBtoY(colorParsley(darkText)), sRGBtoY(testColor))) >= Math.abs(APCAcontrast(sRGBtoY(colorParsley(lightText)), sRGBtoY(testColor)))
}
```

Reply:
Yep, great suggestion!